sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -299,19 +284,21 @@ class MapboxMap :
         transitionOptions?.let(style::setStyleTransition)
       },
       styleDataSourcesLoadedListener = { style ->
-        // TODO https://github.com/mapbox/mapbox-maps-android/issues/1371
-//        styleExtension.sources.forEach {
-//          it.bindTo(style)
-//        }
-//        styleExtension.layers.forEach { (layer, layerPosition) ->
-//          layer.bindTo(style, layerPosition)
-//        }
+        styleExtension.sources.forEach {
+          it.bindTo(style)
+        }
```

Comment:
what would happen if source already exist? Would we skip adding it and continue with adding layers? Or we just throw exception?

Reply:
It will throw an exception, thus we do only call this once after SOURCES event, or, if it's missing - after full onStyleLoaded callback